From c27232c954cdbe0207252ad88fddad4fd6ac0fbc Mon Sep 17 00:00:00 2001
From: Helmut Schaa <helmut.schaa@googlemail.com>
Date: Wed, 8 Jan 2014 13:48:33 +0100
Subject: [PATCH 100/104] netdev-linux: Use unsigned int for ifi_flags

ifi_flags is unsigned, the local equivalents should do the same.

Signed-off-by: Helmut Schaa <helmut.schaa@googlemail.com>
---
 lib/netdev-linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/netdev-linux.c
+++ b/lib/netdev-linux.c
@@ -2810,7 +2810,7 @@ update_flags(struct netdev_linux *netdev
              enum netdev_flags on, enum netdev_flags *old_flagsp)
     OVS_REQUIRES(netdev->mutex)
 {
-    int old_flags, new_flags;
+    unsigned int old_flags, new_flags;
     int error = 0;
 
     old_flags = netdev->ifi_flags;
